### PR TITLE
Update continuous-verification-faqs.md

### PR DESCRIPTION
**Pull request by pras-harness**
Updatede 2 faq

Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \ Faq added under CV

### CODE REVIEW

File: docs/continuous-delivery/verify/continuous-verification-faqs.md [hunk 1]
```diff
@@ -79,4 +79,12 @@ No monitoredServiceRef is defined for service <servicename> and env <envename>
 Can you check if Continuous Verification is selected for the configured monitored service health source under **Assign**.
 You can select the services for which you want to apply the metric.
 
+### Can we add multiple node filters while using Prometheus?
 
+Yes, you can use a multiple-node filter. All of those filters should be applied.
+
+
+
```
Comment (SushrutHarness) [inline docs/continuous-delivery/verify/continuous-verification-faqs.md:87]:
```suggestion
```

File: docs/continuous-delivery/verify/continuous-verification-faqs.md [hunk 2]
```diff
@@ -79,4 +79,12 @@ No monitoredServiceRef is defined for service <servicename> and env <envename>
...
+### How to select the correct Service Instance Identifier (SII )
```
Comment (SushrutHarness) [inline docs/continuous-delivery/verify/continuous-verification-faqs.md:88]:
```suggestion
### How to select the correct Service Instance Identifier (SII)
```

**Review by SushrutHarness — APPROVED** [review]

**Comment by ravilach** [comment]
@SushrutHarness / @krishi0408 mind approving / merging? Thanks!

File: docs/continuous-delivery/verify/continuous-verification-faqs.md [hunk 3]
```diff
@@ -79,4 +79,10 @@ No monitoredServiceRef is defined for service <servicename> and env <envename>
 Can you check if Continuous Verification is selected for the configured monitored service health source under **Assign**.
 You can select the services for which you want to apply the metric.
 
+### Can we add multiple node filters while using Prometheus?
 
+Yes, you can use a multiple-node filter. All of those filters should be applied.
+
+### How to select the correct Service Instance Identifier (SII)
```
Comment (SushrutHarness) [inline docs/continuous-delivery/verify/continuous-verification-faqs.md:86]:
```suggestion
### How do you select the correct Service Instance Identifier (SII)?
```

File: docs/continuous-delivery/verify/continuous-verification-faqs.md [hunk 4]
```diff
@@ -79,4 +79,10 @@ No monitoredServiceRef is defined for service <servicename> and env <envename>
...
+
+It depends upon what kind of metric you want to monitor. For K8S deployments, the best SII selection is typically pod, podname, or containername.
```
Comment (SushrutHarness) [inline docs/continuous-delivery/verify/continuous-verification-faqs.md:88]:
```suggestion
It depends upon what kind of metric you want to monitor. For K8s deployments, the best SII selection is typically pod, podname, or containername.
```

**Review by SushrutHarness — APPROVED** [review]
LGTM

**Review by SushrutHarness — APPROVED** [review]
LGTM

**Pull request merged**